kernel/src/misc/wm/wm.c

```diff
@@ -36,7 +36,16 @@ static uint32_t id_count = 0;
 static fb_t fb;
 static mouse_t mouse;
 
+void buffer_dump(void* buff, size_t len) {
+    for (size_t i = 0; i < len; i++)
+    {
+        printf("%d:%#2X ", i, ((uint8_t*)buff)[i]);
+    }
+    printf("\n");
+}
+
```

Comment:
Remove this, or move it to sys.h (in which case, put the braces on the same line for the for, add a blank line after the for)

Reply:
rather strange that clang_tidy didn't take care of this function ^^